### ISSUE TITLE
A way to install binary version

### ISSUE THREAD

**Issue by FredrikNoren** [AmbientOrg/Ambient#128]
Right now a user can download the binary released version from github releases, but that just gives them a `.exe` to put wherever they like. It would be nice if it would either "install itself"; i.e. you could run `ambient --install` and it would put itself somewhere where it makes sense and add PATH variables for it, or if we had some kind of installer script/app that does it.

**Comment by philpax** [comment]
`cargo-dist` has experimental installers as part of its offering (which you can see [here](https://github.com/axodotdev/cargo-dist/releases/tag/v0.0.2)) but we probably want to wait a bit more, still.

**Comment by SK83RJOSH** [comment]
If this ever gets realized, I would like to add that it would be pretty nice to have options to install pre-requisites (VS2022 + payloads; rustup + nightly) which would help alieviate #126. In the meantime something similar to Unreal's Setup.bat/sh/command may be a good idea.

**Comment by philpax** [comment]
We're planning on removing the auto-rustc-download stuff, so getting rustc set up would be your own problem. We'll provide advice around that - especially because you don't need nightly - but it is something to be aware of.

**Comment by FredrikNoren** [comment]
Another note; long term we probably will want to split ambient up into two applications; the runtime and the "version manager". What's currently `ambient` would become `ambient_runtime`, and then we introduce a new app; `ambient`, which will download the correct `ambient_runtime` based on the version in the `ambient.toml` (exactly how rust and cargo works).

**Comment by philpax** [comment]
With that, I think it'd stay `ambient` - the Rust approach has `rustup`, which just changes what `rust` points to instead of actually switching out the executable. So probably `ambient` and some wordplay for switching ambience?

**Comment by philpax** [comment]
@FredrikNoren I know you were experimenting with binary build installations recently; do you have an update for this or are you still working on it?

**Comment by FredrikNoren** [comment]
Yeah I'm working on adding it to webinstall: https://github.com/webinstall/webi-installers/pull/626 . There's a DNS issue blocking us from testing it on windows, waiting for them to respond on that.
